portal/docs/overview/report_payments.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 4
+---
+
+# Using the API to report repayments
+
+We recommend that your technical team consults with the Kiva Coordinator at your organization to fully understand Kiva's Repayment Reporting process. To do so, please have your technical team review the following:
+
+This video provides a thorough explanation of the repayment reporting process
```

Comment:
yikes.. the link that is on the Zendesk site points to a private video. I wonder why.. we should try and get an updated link or remove it. Probably an Editoral Stage thing, also.

https://www.youtube.com/watch?v=wPlx30TgV6k